node/src/actors/messages.rs

```diff
@@ -169,6 +170,8 @@ pub struct BuildVtt {
     pub vto: Vec<ValueTransferOutput>,
     /// Fee
     pub fee: u64,
+    /// Strategy to sort the unspent outputs pool
+    pub utxo_strategy: UtxoSelectionStrategy,
```

Comment:
We can add a serde attribute to deserialize as the default if the value is missing, this way we don't break compatibility, if someone was using the JSON-RPC send method it will keep working the same way as before.

I think you will have to implement Default as UtxoSelectionStrategy::Random.

```suggestion
    #[serde(default)]
    pub utxo_strategy: UtxoSelectionStrategy,
```